### PR TITLE
Update zlib.py

### PR DESCRIPTION
**Pull request by Caesurus**
according to https://tools.ietf.org/id/draft-levine-application-gzip-03.html#rfc.section.2 the mime-type `application/zlib` is also valid, and so should be handled by this plugin.

### CODE REVIEW

File: fact_extractor/plugins/unpacking/linuxkernel/test/test_plugin_linuxkernel.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class TestPaToolUnpacker(TestUnpackerBase):
+
+    def test_unpacker_selection_generic(self):
+        self.check_unpacker_selection('linux/kernel', 'LinuxKernel')
+
+    def test_extraction_valid_bzImage(self):
+        input_file = Path(TEST_DATA_DIR, 'bzImage')
+        files, meta_data = self.unpacker.extract_files_from_file(str(input_file), self.tmp_dir.name)
+
+        print('files returned in test: ', files)
```
Comment (jstucke) [inline fact_extractor/plugins/unpacking/linuxkernel/test/test_plugin_linuxkernel.py:24]: you probably want to remove the `print()`

**Review by jstucke — COMMENTED** [review]
Hi, thank you very much for your contribution. The pull request looks already mostly fine. There are some coding guidelines that we agreed to internally (e.g. using pathlib instead of os.path). It would be great if you could edit those parts but it would also be alright if you choose not to. I was able to confirm, that the unpacking works and the test coverage looks also great.

**Comment by Caesurus** [comment]
Shoot, I had _meant_ to do two PRs, one for the zlib mime-type, and then another separate one for the linux kernel stuff, but silly me... i forgot to branch off and kept working in `master`.

I have some changes i haven't pushed yet. and I'll incorporate your review comments as well. I'll As such, I'll convert this to draft for the time being. Sorry for the mixup :)

**Comment by Caesurus** [comment]
Ok, should be ok now. I went and compiled a kernel as small as i easily could (`make tinyconfig`) and made a bzImage for each compression supported in the kernel menuconfig. I checked those in as test data, so there is better coverage of the different compressions (and making sure the tools are actually installed correctly). I don't know how you feel about adding that many binary files to the repo. If it's frowned upon, I can remove a bunch.

**Review by jstucke — APPROVED** [review]
Looks fine. The CI built was also successful. Thanks again for contribution!

**Pull request merged**